snmp/datadog_checks/snmp/data/default_profiles/barracuda-cloudgen.yaml

```diff
@@ -8,7 +8,7 @@ metrics:
   - MIB: PHION-MIB
     table:
       OID: 1.3.6.1.4.1.10704.1.0
-      name: boxServices
+      name: firewall
```

Comment:
This is a false positive, `1.3.6.1.4.1.10704.1.0` refers to boxServices

The tooling is stripping .0 and leads to `1.3.6.1.4.1.10704.1`  that refers to firewall.